### ISSUE TITLE
[FEATURE] Introduce storybook

### ISSUE THREAD

**Issue by fekete965** [vatsalsinghkv/easy-fix#52]
### Description

Adding storybook to the project would let us build, manage and easily test our UI components.
The changes can also be deployed to a preview to observe them.
That would also allow people to create separate components first where people can review and play with the components before an integrating it into the website.

### ✨ Browser

Google Chrome

### Checklist 🚀

- [X] I checked and didn't find similar [issue](https://github.com/vatsalsinghkv/easy-fix/issues)
- [X] I have read the [Contributing Guidelines](https://github.com/vatsalsinghkv/easy-fix/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (blank for no).

**Comment by vatsalsinghkv** [comment]
@fekete965 I was thinking about this for having a proper consistent design system

**Comment by fekete965** [comment]
@vatsalsinghkv Yes, it would be beneficial and it would open up opportunities for the community to share a component in an isolated environment without tinkering with the application itself. It would be super easy to test and then swap any component without potentially breaking anything.
Since we talked about moving the project to next.js (or something similar) I will make the components compatible for both client and server.
